src/main/java/io/redisearch/Schema.java

```diff
@@ -47,6 +47,11 @@ public void serializeRedisArgs(List<String> args) {
                 args.add("NOINDEX");
             }
         }
+
+        @Override public String toString() {
+            return "Field{" + "name='" + name + '\'' + ", type=" + type + ", sortable=" + sortable + ", noindex=" + noindex
```

Comment:
You should try to avoid unnecessary string concats where possible 
e.g.
`"Field{" + "name='"` -->  `"Field{name='"`